src/System.Runtime/tests/System/Reflection/PointerTests.cs

```diff
@@ -184,6 +198,15 @@ public void PointerMethodParameter(int value)
             method.Invoke(obj, new[] { Pointer.Box(unchecked((void*)value), typeof(byte*)), value });
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Pointers through Invoke not implemented: https://github.com/dotnet/corert/issues/2113")]
+        public void PointerNullMethodParameter()
+        {
+            var obj = new PointerHolder();
+            MethodInfo method = typeof(PointerHolder).GetMethod("Method");
+            method.Invoke(obj, new object[] { null, 0 });
```

Comment:
Is there anything to assert, or we're just verifying it doesn't throw/crash?

Reply:
The body of the invoked method has `Assert.Equal(expected, unchecked((int)ptr));`